dmutils/email/dm_notify.py

```diff
@@ -182,12 +198,14 @@ def send_email(
                 )
 
         except HTTPError as e:
-            self._log_email_error_message(to_email_address, template_name_or_id, reference, e)
+            self._log_email_error_message(email_obj, e)
             if isinstance(e.message, list) and \
                     any(msg["message"].startswith("Missing personalisation") for msg in e.message):
                 raise EmailTemplateError(str(e))
             raise EmailError(str(e))
 
+        self._log(logging.INFO, "Email sent to Notify successfully", email_obj)
```

Comment:
What does this look like on the command line when being run as part of a script? Can we see the email object's fields as well as the message?

Reply:
🤔 I think not, but I'll get back to you on that

Reply:
:+1: If not, might it be worth adding the `reference` to the message?

Reply:
Yeah that's probably a good idea whatever the case; the handling of the `extra` field depends on the formatter and so could be different in different circumstances.